scripts/start.sh

```diff
@@ -33,7 +33,7 @@ INSTALLED_DOCKER_VERSION=$(docker version -f "{{.Server.Version}}")
 MINIMUM_REQUIRED_DOCKER_VERSION_REGEX="^((([2-9][1-9]|[3-9][0]|[0-9]{3,}).*)|(20\.([0-9]{3,}|[1-9][1-9]|[2-9][0]).*)|(20\.10\.([0-9]{3,}|[2-9][0-9]|[1][3-9])))"
 if [[ $INSTALLED_DOCKER_VERSION =~ $MINIMUM_REQUIRED_DOCKER_VERSION_REGEX ]]; then
     if command -v gp version &> /dev/null; then
-        export PROXY_TLS="-DgitPodEnvironment"
+        export SENAITE_GITPOD_HTTPS_REWRITE="true"
     fi
```

Comment:
Since we support both Traefik and Nginx, should we be checking if Traefik is enabled?

Reply:
We've not yet run into issues with Traefik especially that we never run in with Gitpod. I'd worry about Traefik when its use within Gitpod shows up. We basically use traffic in prod.

Reply:
I think the latest commit fixes the issue, https://github.com/ozone-his/ozone-docker-compose/pull/75/commits/1b11d6bd9df940285e99d19a111b23955e5e6836 .

Reply:
Could you try again? Thanks.